include/openenclave/attestation/verifier.h

```diff
@@ -165,6 +165,44 @@ oe_result_t oe_verify_evidence(
     oe_claim_t** claims,
     size_t* claims_length);
 
+/**
+ * Type definition for a claims verification callback.
+ *
+ * @param[in] claims a pointer to an array of claims
+ * @param[in] claims_length length of the claims array
+ * @param[in] arg caller defined context
+ */
+typedef oe_result_t (*oe_verify_claims_callback_t)(
+    oe_claim_t* claims,
+    size_t claims_length,
+    void* arg);
+
+/**
+ * oe_verify_attestation_certificate_with_evidence
```

Comment:
As before, I think this name should change.  e.g., `oe_verify_self_signed_certificate_with_evidence` or something similar.